openarc/openarc.c

```diff
@@ -1406,6 +1406,8 @@ arcf_config_load(struct config *data, struct arcf_config *conf,
 			conf->conf_signalg = arcf_lookup_strtoint(str,
 			                                          arcf_signalgorithms);
 		}
+		else
+			conf->conf_signalg = ARC_SIGN_RSASHA256;
```

Comment:
Add braces around this, please.  (Old coding standard rule, need to be consistent.)